packages/backend/src/models/SearchModel/utils/search.ts

```diff
@@ -45,6 +45,21 @@ export function getRegexFromUserQuery(query: string) {
     return new RegExp(splitQuery.join('|'), 'ig');
 }
 
+export function getColumnMatchRegexQuery(
+    queryBuilder: Knex.QueryBuilder,
+    searchQuery: string,
+    columns: string[],
+) {
+    const regex = getRegexFromUserQuery(searchQuery).source;
+
+    // use regexp_matches
+    return queryBuilder.where((builder) => {
```

Comment:
I'm wondering how efficient this regex is...

Reply:
This regex just splits the words in the query by spaces and then joins them using an OR conditional operator `|` so it needs to match full words in your search query - also it is case insensitive

I believe it should be fine as people will be searching just by name, email, role

Btw this is already being used to find matching tables and fields in the global search